CHANGES_UNRELEASED.md

```diff
@@ -15,3 +15,16 @@
   Applications that submit telemetry via Glean must request a data review for these metrics
   before integrating the places component. See the component README.md for more details.
   ([#2431](https://github.com/mozilla/application-services/pull/2431))
+
+## FxA Client
+
+### Breaking Changes
+
+* `migrateFromSessionToken` now returns a metrics JSON object if the migration succeeded.
+  ([#2492](https://github.com/mozilla/application-services/pull/2492))
+
+### Features
+
+* `migrateFromSessionToken` now handles offline use cases. It caches the data the consumers originally provide.
+  If there's no network connectivity then the migration could be retried using the new `retryMigrateFromSessionToken` method. 
```

Comment:
It's not clear from the description here exactly *how* the consumer should know whether or not to call this method (assuming that the migration could fail for some reasons that can be retried, and some reasons that cannot).